be/CMakeLists.txt

```diff
@@ -211,6 +211,9 @@ message(STATUS "Using OpenSSL Root Dir: ${OPENSSL_ROOT_DIR}")
 add_library(crypto STATIC IMPORTED)
 set_target_properties(crypto PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libcrypto.a)
 
+add_library(AWS::crypto STATIC IMPORTED)
```

Comment:
Why did you add this library? I don't see how it is used?

Reply:
After aws-sdk-cpp is upgraded from 1.9.179 to 1.0.272, it will fail with the following error when building thirdparty libraries:
```
CMake Error at src/service/CMakeLists.txt:49 (add_executable):
  Target "starrocks_be" links to target "AWS::crypto" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
```
My solution is to add library: AWS::crypto. But if there is other better approches, I will be very happy to learn and adopt.

Reply:
Did you try to add crypto in find package of awssdk?

Reply:
Let me try it. Thanks!